aries-site/src/components/accessibility/Video.js

```diff
@@ -0,0 +1,50 @@
+import React, { useContext } from 'react';
+
+import { Anchor, AnnounceContext, Box } from 'grommet';
+
+export const AccessVideo = () => {
+  const announce = useContext(AnnounceContext);
+
+  return (
+    <>
+      <Box 
+        gap="small" 
+        width="large" 
+        pad={{ bottom: 'large' }}
+      >
+        <div id="video-alt" style={{ display: 'none' }}>
```

Comment:
I would have expected this div to be nested in the `<video>` tag after the `<track>`. Is there a reason for having it outside?

Reply:
When testing this myself and Bill with the div inside the `<video />`, the information wasn't being read aloud by the screen reader. It was only relayed with this div being outside of the `<video />`.

For my most recent changes, you will see that I am using `aria-description` instead of `aria-describedby`, as Bill was still having some issues with getting the description read aloud using JAWS. They serve the same purpose, but have different levels of support/implementation. Using `aria-description` works for me on VO and I am hoping Bill has a better experience with JAWS.

Reply:
For my education, can you share any resources speaking to `aria-description` vs `aria-describedby`? I'd like to better understand why `aria-describedby` wasn't working. Is is fundamentally flawed? Was it a flaw in our implementation? It is so widely referenced and documented, that it seems like we should deeply understand this so that we are aware of pitfalls and can recognize and abate them.